src/Illuminate/Container/CircularReferenceException.php

```diff
@@ -0,0 +1,36 @@
+<?php namespace Illuminate\Container;
+
+use Exception;
+
+class CircularReferenceException extends Exception {
+
+	/**
+	 * @var array
+	 */
+	protected $buildStack;
+
+    /**
+     *  Create a new Circular Reference Exception instance
```

Comment:
When are we going to finally follow PSR-2... :mask: 

About the description, it was supposed to be CircularReferenceException. Should I change this?